server/datastore/mysql/apple_mdm.go

```diff
@@ -1245,10 +1284,14 @@ func (ds *Datastore) BulkSetPendingMDMAppleHostProfiles(ctx context.Context, hos
 			uuids = hostUUIDs
 
 		case len(hostIDs) > 0:
+			// TODO: if a very large number (~65K) of uuids was provided, could
+			// result in too many placeholders (not an immediate concern).
```

Comment:
Eventually we should do a thorough review of statements expanded with `sqlx.In` or dynamically with `strings.Repeat` to prevent any other issues. We may even be able to extract helper functions and streamline the handling of those batches. For now I went with the critical ones that we know we may hit.